calico_upgrade/pkg/calicoupgrade.go

```diff
@@ -31,14 +31,12 @@ func main() {
     validate  Validate the v1 formatted data, check that it can be migrated to
               the v3 format, and output a full report of any migrated names,
               migration errors, or migrated name conflicts.
-    start     Start the upgrade process.  This will validate the data, pause
-              Calico networking, and migrate the data from v1 to v3 format.
+    start     Start the upgrade process. This validates the data, pauses
+              Calico networking, and migrates the data from v1 to v3 format.
               Once the data is migrated successfully, the calico/node
               instances can be upgraded to v3.x.
-    complete  This completes the upgrade process by un-pausing Calico
-              networking.
-    abort     This aborts the upgrade process by un-pausing Calico
-              networking.
+    complete  Complete the upgrade process by un-pausing Calico networking.
```

Comment:
Not sure that "un-pausing" is a word.... I would actually suggest that with the command being `complete`, you could omit that as prefatory content (somewhat redundant). So, I think you could just say "Resumes Calico networking."

Reply:
indeed :-)   I was tired when i wrote it lol

I've modified this to resume in my PR which I haven't put up for review yet...but Brendan feel free to fix it too

Reply:
However, since both "complete" and "abort" seem to do nothing except resume Calico networking (ostensibly), I think something would be missing if we shortened these descriptions to that degree. Therefore, I am just going out on a limb here, but I might suggest:
- "Complete the upgrade and resume Calico networking"
- "Abort the upgrade and resume Calico networking"
Again, I am not sure just what is involved in `complete` and `abort` beyond resuming Calico networking. It might be nice to include something about that, as to just what it is (to be more specific in the descriptions), because I can't imagine we would need two different commands which both did the same thing?

Reply:
Reading further down below, it seems that perhaps it is not "Calico networking" per se that's resumed, but rather, the ability to create new endpoints. If so, I think it would be better to rephrase as 
- "Complete the upgrade and allow new endpoint creation"
- "Abort the upgrade and allow new endpoint creation"

If you just say "Calico networking" then people might think they will lost connectivity, which appears to not be the case.